### PR TITLE
Include mission statement on the home page

### PR DESCRIPTION
**Pull request by Relequestual**
Resolves https://github.com/json-schema-org/community/issues/97

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -5,8 +5,11 @@ permalink: /
 ---
 
 
-**JSON Schema** is a vocabulary that allows you to **annotate** and **validate** JSON documents.
+**JSON Schema** enables the confident and reliable use of the JSON data format.
```
Comment (Julian) [inline index.md:8]: I think the original language here was a bit more concrete (i.e. corresponded directly to specific functionality). So maybe we leave both? E.g.

```suggestion
**JSON Schema** is a vocabulary that allows you to **annotate** and **validate** JSON documents. It enables the confident and reliable use of the JSON data format.
```

File: index.md [hunk 2]
```diff
@@ -5,8 +5,11 @@ permalink: /
 ---
 
 
-**JSON Schema** is a vocabulary that allows you to **annotate** and **validate** JSON documents.
+**JSON Schema** is a vocabulary that allows you to **annotate** and **validate** JSON documents. It enables the confident and reliable use of the JSON data format.
```
Comment (jdesrosiers) [inline index.md:8]: I don't think "vocabulary" is the right word here. Maybe something more like this?

```suggestion
**JSON Schema** is a JSON document that allows you to define rules to **annotate** and **validate** JSON documents. It enables the confident and reliable use of the JSON data format.
```

Comment (Relequestual) [reply index.md:8]: I would argue that "**A** JSON Schema" is a "JSON document", while "JSON Schema" the specification defines a vocabulary for use within "a JSON Schema JSON document"

Comment (Relequestual) [reply index.md:8]: Also, this is not something that has changed in this PR.
This PR adds to that line, but mainly it's about the additional "mission" section.

Comment (jdesrosiers) [reply index.md:8]: I'm fine if you don't want to change this now. It's just something that bothers me every time I see it and this seemed like a good opportunity to fix it. What does "vocabulary" even mean in this context? I'm pretty sure it has nothing to do with JSON Schema Vocabularies, but I can't be sure. It's just way too generic a word to be meaningful.

As I seem to be saying a lot lately, I like to describe JSON Schema as a declarative domain specific language for validating and annotating JSON-compatible data. I suggest we used those words, but whatever you choose is fine even if you choose to do nothing for now and come back to it later.

Comment (Julian) [reply index.md:8]: It's being used in a non technical sense I suspect. And very likely predates any technical thing called vocabularies in JSON Schema. It doesn't bother me a ton but I agree it doesn't seem like the best way to describe us either so I'm definitely not opposed to changing it either.

File: index.md [hunk 3]
```diff
@@ -5,8 +5,11 @@ permalink: /
...
 
+## Mission
+
+Our mission is to enable the definition of interoperable contracts for JSON data in the form of schemas, using constraints based validation rules, while enabling and supporting use cases beyond validation.
```
Comment (Julian) [inline index.md:12]: I find this more natural without the second comma -- 

```suggestion
Our mission is to enable the definition of interoperable contracts for JSON data in the form of schemas, using constraints based validation rules while enabling and supporting use cases beyond validation.
```

Content-wise it seems reasonable enough to me though so happy to thumbs up myself if the plan is to address Jason's point above in some other PR later.

**Review by Julian — APPROVED** [review]

File: index.md (hunk 3 above)
Comment (Relequestual) [reply index.md:12]: An editor friend of mine has confirmed that the comma is necessary. Otherwise the meaning sort of changes.

**Comment by awwright** [comment]
This strikes me as a bit wordy. It doesn't help me understand if I'm the target audience or not.

> declarative domain specific grammar and syntax

This is too many adjectives for me. Describing the technical basis of JSON Schema is a good idea, but they can be split out into a separate sentence that defines the lingo. For example

> JSON Schema is declarative, where you make assertions about what JSON is expected to look like.

"domain specific" is redundant (we say it's only for JSON, that's it's "domain"). And "grammar and syntax" I don't think is accurate (the grammar is JSON, we don't define our own grammar, we define an existing grammar's semantics).

> JSON Schema enables the confident and reliable use of the JSON data format.

This doesn't help me understand if I'm the target audience or not. If we absolutely have to, describe the impact on actual developers, like "Developers use JSON Schema as a reliable alternative to hard-coding validation rules in code."

File: index.md [hunk 4]
```diff
@@ -5,8 +5,11 @@ permalink: /
 ---
 
 
-**JSON Schema** is a vocabulary that allows you to **annotate** and **validate** JSON documents.
+**JSON Schema** is a declarative domain specific grammar and syntax that allows you to **annotate** and **validate** JSON documents. JSON Schema enables the confident and reliable use of the JSON data format.
```
Comment (jdesrosiers) [inline index.md:8]: Isn't "grammar and syntax" just a more verbose way to say "language"? Also, domain specific language (DSL) is specific well defined technical term. Saying "language" in a different way shouldn't change the meaning, but it does increase the cognitive load of parsing this sentence. Of course it could just be that the sentence is easy to parse because I'm familiar with these technical terms. Not everyone is.

Comment (awwright) [reply index.md:8]: JSON Schema is way closer to a language than a grammar or syntax. Our grammar and syntax is entirely imported, and in no way domain-specific.

> Not everyone is.

Most of our intended audience should be familiar with the terms, but our front page should be written for the widest possible audience, not just our intended audience.

**Comment by jdesrosiers** [comment]
> "domain specific" is redundant  (we say it's only for JSON, that's it's "domain")

The domain is JSON validation and annotation, but I agree that "domain specific" is not useful for anyone who isn't familiar with the concept of a domain specific language (DSL). Would this be better?

> JSON Schema is a declarative language that allows you to **annotate** and **validate** JSON documents.

**Comment by awwright** [comment]
> JSON Schema is a declarative language that allows you to **annotate** and **validate** JSON documents.

💯

**Comment by awwright** [comment]
The only reason I would prefer "vocabulary" or some similar term over "language" is because "language" to me includes both the grammar/syntax and the vocabulary. JSON Schema does not define any syntax, only the vocabulary.

But it also seems to me "language" is a more accessible term than "vocabulary", here.

File: index.md (hunk 4 above)
Comment (Relequestual) [reply index.md:8]: Let's go with language then!

File: index.md [hunk 5]
```diff
@@ -5,8 +5,11 @@ permalink: /
...
 
+## Mission
+
+Our mission is to enable the definition of interoperable contracts for JSON data in the form of schemas, using constraints based validation rules, while enabling and supporting use cases beyond validation.
```
Comment (handrews) [inline index.md:12]: I would like to add something to this that mentions annotation by name, because the spec should ensure interoperability around producing and consuming annotations.  The phrase "supporting use cases beyond validation" is vague and completely open-ended.  Annotations are _how_ additional use cases are supported, and that's what the JSON Schema project is concerned with.

Perhaps just this would do:

> Our mission is to enable the definition of interoperable contracts for JSON data in the form of schemas, using constraints-based validation rules, while enabling and supporting **_further use cases through annotation._**

I think that makes it clear that both validation and annotation are part of the interoperable contracts.

Comment (Relequestual) [reply index.md:12]: Good point. Thinking on this though, I consider the use case where a schema is taken "as is" without an instance, and something is created, like a class or a database table (or 2). Are they still annotations?

I wonder if there's a new class of keywords for new vocabularies which are intended to be used without an instance in mind... maybe "declaration". I guess that could be seen as we're saying it's a "declarative" language. We have the duality being used both with and without an instance. Without an instance, it sort of becomes declarative, but still constraints based.

Comment (handrews) [reply index.md:12]: > Thinking on this though, I consider the use case where a schema is taken "as is" without an instance, and something is created, like a class or a database table (or 2). Are they still annotations?

Ooooh... excellent question!  Yeah, I've tended to think of that as a different way of making use of the existing keyword classes (not just annotations - assertions like `type` and applicators like `properties` are part of such cases as well).  But that way of declaring things with a schema is worth calling out explicitly.  I think of these as "generative" use cases (generating an instance from a schema- not necessarily generating a JSON instance but usually a code generator or whatever is still working with the JSON data model which is translated further into Java, SQL, etc.

Definitely worth further thought and clear language.

**Review by handrews — CHANGES_REQUESTED** [review]
I'm marking this "request changes" because the discussions seem to have stalled with several open questions despite the approval from earlier.

**Comment by Relequestual** [comment]
While the first part of the change makes senes, I'm not sure the "mission" part does right now.

I'll move the first part of this PR to a new PR, and close this PR, in favour of further investigation on vision and planning with a deliverable being a charter and mission statement.

**Comment by Relequestual** [comment]
Closed in favour of https://github.com/json-schema-org/json-schema-org.github.io/pull/487
